apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/TypeName.kt

```diff
@@ -0,0 +1,89 @@
+package com.apollographql.apollo3.compiler
+
+import com.apollographql.apollo3.compiler.codegen.java.JavaAnnotations
+import com.apollographql.apollo3.compiler.codegen.java.JavaClassNames
+import com.squareup.javapoet.ClassName
+import com.squareup.javapoet.CodeBlock
+import com.squareup.javapoet.ParameterizedTypeName
+import com.squareup.javapoet.TypeName
+import com.squareup.javapoet.WildcardTypeName
+
+fun TypeName.isList() =
```

Comment:
These extension methods were ported from [apollo/compiler/testUtils.kt](https://github.com/apollographql/apollo-kotlin/blob/66cd44629e374b667d16ee831f5a4d42cc38c50b/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/testUtils.kt)

Reply:
Can you make it `internal` ?

Reply:
Makes sense. None of the original methods were internal, and I don't imagine these extensions methods would get used outside of the compiler, but might as well do it.

Reply:
Exactly! This is not a huge issue at the moment but the compiler might become transitively included more and more and we don't want to pollute the namespace too much.